scripts/build_qrview.js

```diff
@@ -0,0 +1,19 @@
+const fs = require("fs");
+const path = require("path");
+
+const jsqrPath = path.join(__dirname, "../node_modules/jsqr/dist/jsQR.js");
```

Comment:
Can we do this using `require.resolve` or something similar? So it's more dynamic?

Reply:
Done, actually i didn't know about `require.resolve`.